scripts/common/repository.js

```diff
@@ -112,46 +110,107 @@ class Repository {
     });
   }
 
+  static fetchTags() {
+    return execa.shell('git describe --abbrev=0').then(({ stdout }) => stdout).catch(() => '');
+  }
+
+  createSmokeTag(tag) {
+    return execa.shell(`git tag ${this.name}@${this.version} ${tag}`);
+  }
+
+  deleteSmokeTag() {
+    return execa.shell(`git tag -d ${this.name}@${this.version}`);
+  }
+
   updateChangelog() {
-    const fetchTags = () => execa.shell('git describe --abbrev=0').then(({ stdout }) => stdout).catch(() => '');
-    const createSmokeTag = tag => execa.shell(`git tag ${this.name}@${this.version} ${tag}`);
-    const deleteSmokeTag = () => execa.shell(`git tag -d ${this.name}@${this.version}`);
     const getChangelog = () => execa.shell(`lerna exec --scope ${this.name} --concurrency 1 --no-sort -- conventional-changelog \
       --preset angular --lerna-package ${this.name} --commit-path ${this.location}`);
     const updateChangelog = () => execa.shell(`lerna exec --scope ${this.name} --concurrency 1 --no-sort -- conventional-changelog \
       --preset angular --infile CHANGELOG.md --same-file --lerna-package ${this.name} --commit-path ${this.location}`);
-    return fetchTags()
-      .then(createSmokeTag)
+    return this.constructor.fetchTags()
```

Comment:
```suggestion
    return Repository.fetchTags()
```
not mandatory but i think it's easier to read